initramfs-init.in

```diff
@@ -646,6 +652,22 @@ if [ -n "$ovlfiles" ]; then
 else
 	apk add --root $sysroot $repo_opt $apkflags $pkgs
 fi
+
+if [ -n "$ssh_install" ]; then
+	rc_add sshd default
+	echo "PermitRootLogin yes" >> $sysroot/etc/ssh/sshd_config
+	if [ -n "$ssh_key" ]; then
+		mkdir -p $sysroot/root/.ssh
+		wget "$ssh_key" -O $sysroot/root/.ssh/authorized_keys
+	elif [ -n "$ssh_pass" ]; then
```

Comment:
we shouldnt use `elif` here. I think it should be possible to set both `ssh_key` and `ssh_pass` if that is wanted.